src/tags/object/Image.js

```diff
@@ -57,6 +58,7 @@ const TagAttrs = types.model({
   height: types.maybeNull(types.string),
   maxwidth: types.optional(types.string, "100%"),
   maxheight: types.optional(types.string, "calc(100vh - 194px)"),
+  smoothing: types.maybeNull(types.boolean),
```

Comment:
Is this not an optional with a default as is stated in the jsdoc? I think it would make the logic a bit more clear if it was as the above typecheck is not required in this case.

Reply:
we have similar behaviour for `showLabels` parameter and setting — if it's not set we use settings, if it's set we use this value from config. so actually it looks good, but jsdoc definition shouldn't have default value, and it's better to add that by default it uses user settings. @juliosgarbi

Reply:
done!